docs/can-guides/experiment/setting-up-canjs.md

```diff
@@ -284,3 +284,9 @@ Another quick way to start locally is by loading CanJS from a CDN:
     </body>
     </html>
 @highlight 6-6
+
+## A note on Promises
+
+Native [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) are used in CanJS, which means you need to:
+
+* Include a Promise polyfill if targeting browsers that do not have native support. [Steal](http://stealjs.com/) includes a polyfill for you.
```

Comment:
It seems odd that this is a list but there’s only one list item. I would include this sentence with the paragraph above to make it flow more naturally.

Also, it’d be nice to mention which browsers this applies to (I know at least IE9, not sure which versions of others).